### PR TITLE
Add disposableWebPage

### PR DESCRIPTION
**Pull request by Anshuman-Verma**
**http://disposablewebpage.com/ `[Account]`**

**Create a temporary web page that self-destructs.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [Google Structured Data Testing Tool](https://developers.google.com/structured-data/testing-tool/) - Test a webpage for structured data and SEO.
 * [Regulex](http://jex.im/regulex/) - JavaScript regular expression visualizer.
 * [GTmetrix](https://gtmetrix.com/) - Analyze your site’s speed and make it faster.
+* [disposableWebPage](http://disposablewebpage.com/) `[Account]` - Create a temporary web page that self-destruct.
```
Comment (aviaryan) [inline README.md]: that *will* self-destruct
OR
that *self-destructs*

Comment (Anshuman-Verma) [inline README.md]: @aviaryan copied that :fearful:

**Comment by Anshuman-Verma** [comment]
@aviaryan review again.

**Comment by Anshuman-Verma** [comment]
Fixed PR comment.

**Comment by spekulatius** [comment]
gets discontinued shortly: https://disposablewebpage.com/notice-of-product-discontinuation/

**Comment by aviaryan** [comment]
Thanks for letting us know @spekulatius . In that case, I think it's best it is removed from the list. Reverting this PR..

**Comment by spekulatius** [comment]
no worries, just stumbled across this while seeing whats coming into the awesome list :)

**Comment by Anshuman-Verma** [comment]
@spekulatius thanx for bringing that to notice, i totally missed that while creating a PR. :smile:

**Pull request merged**